core/conn/odbc/src/odbc/nsksrvr/SrvrConnect.cpp

```diff
@@ -1190,10 +1176,6 @@ ImplInit (
    srvrGlobal->receiveThrId = getpid();
    timer_register();
 
```

Comment:
Thanks for the change Selva. 
Could you please undo the change for StatisticsTimerHandle ? Thanks

Reply:
Thanks for catching this inadvertent deletion of timer initialization call.